.yamllint

```diff
@@ -11,5 +11,4 @@ rules:
   indentation:
     indent-sequences: consistent
     spaces: 2
-  key-ordering: enable
```

Comment:
I'd love for us to keep sorting of keys in YAML files. Makes it much easier to find things, especially in bigger files.

Reply:
This is not very practical when dealing with Kubernetes manifest YAMLs :)

Reply:
How's that? When I look at `kubectl get deployments crate-operator -o yaml`, the keys are sorted alphabetically. Exactly the way `key-ordering` enforces them.

Reply:
Well, it's the same reason why `black` fails to make properly **human** readable Python code ;)

I usually aim for "easy to read for humans", e.g. compare:

```yaml
    spec:
      containers:
      - name: crate-operator
        image: "crate/crate-operator:latest"
        imagePullPolicy: Always
        env:
        - name: CRATEDB_OPERATOR_BOOTSTRAP_TIMEOUT
          value: "1800"
        - name: CRATEDB_OPERATOR_DEBUG_VOLUME_SIZE
          value: "128GiB"
        - name: CRATEDB_OPERATOR_DEBUG_VOLUME_STORAGE_CLASS
          value: "default"
        - name: CRATEDB_OPERATOR_JMX_EXPORTER_VERSION
          value: "0.6.0"
        - name: CRATEDB_OPERATOR_LOG_LEVEL
          value: "INFO"
        - name: CRATEDB_OPERATOR_ROLLING_RESTART_TIMEOUT
          value: "3600"
        - name: CRATEDB_OPERATOR_SCALING_TIMEOUT
          value: "3600"
```

vs.

```yaml
    spec:
      containers:
      - env:
        - name: CRATEDB_OPERATOR_BOOTSTRAP_TIMEOUT
          value: "1800"
        - name: CRATEDB_OPERATOR_DEBUG_VOLUME_SIZE
          value: "128GiB"
        - name: CRATEDB_OPERATOR_DEBUG_VOLUME_STORAGE_CLASS
          value: "default"
        - name: CRATEDB_OPERATOR_JMX_EXPORTER_VERSION
          value: "0.6.0"
        - name: CRATEDB_OPERATOR_LOG_LEVEL
          value: "INFO"
        - name: CRATEDB_OPERATOR_ROLLING_RESTART_TIMEOUT
          value: "3600"
        - name: CRATEDB_OPERATOR_SCALING_TIMEOUT
          value: "3600"
        image: "crate/crate-operator:latest"
        imagePullPolicy: Always
        name: crate-operator
```

Well, that's my few cents ;-)